compiler/tests-full/stdlib.cma.expected.js

```diff
@@ -497,7 +497,7 @@
      i =  /*<<stdlib.ml:285:2>>*/ 0;
     for(;;){
       /*<<stdlib.ml:280:4>>*/ if(l <= i)
-       /*<<stdlib.ml:280:19>>*/ return s1 + ".";
+       /*<<stdlib.ml:213:2>>*/ return s1 + ".";
```

Comment:
Why do we now see the location of the body of string concat ?

Reply:
I was wondering whether to keep the event at the beginning of the function. But we don't really need it if the function is inlined. And if the function is not inlined, the Chrome debugger is going to skip over it if there is not event, which is probably what we want as well.